src/helm/benchmark/scenarios/wmt_14_scenario.py

```diff
@@ -0,0 +1,95 @@
+from .scenario import Scenario, Instance, Reference, TRAIN_SPLIT, VALID_SPLIT, TEST_SPLIT, CORRECT_TAG, Input, Output
+
+MAX_TRAIN_INSTANCES = 20_000  # This is arbitrary, but 20,000 training examples should be enough.
+
+
+class WMT14Scenario(Scenario):
+    """
+    The 2014 Workshop on Statistical Machine Translation:
+    https://aclanthology.org/W14-3302.pdf
+
+    The scenario consists of 5 subsets, each of which is a parallel corpus between English and another language. The
+    non-English languages include Czech, German, French, Hindi, and Russian.
+
+    For each language pair, the validation and test set each includes around 3,000 examples, while the training set is
+    usually much larger. We therefore randomly downsample the training set to speedup data processing.
+
+    Task prompt structure:
+
+        Translate {source_language} to {target_language}:
+        {Hypothesis} = {Reference}
+
+    Example from WMT14 Fr-En:
+
+        Hypothesis: Assemblée générale
+        Reference: General Assembly
+    """
+
+    name = "WMT_14"
+    description = "Scenario for the 2014 Workshop on Statistical Machine Translation"
+    tags = ["machine_translation"]
+
+    def __init__(self, source_language, target_language):
+        super().__init__()
+        valid_languages = set(["cs", "de", "fr", "hi", "ru", "en"])
+        self.source_language = source_language
+        self.target_language = target_language
+        # The source/target langauges must be included in WMT14
+        assert self.source_language in valid_languages and self.target_language in valid_languages
+        # The source language and the target language should be different
+        assert self.source_language != self.target_language
+        # One of the languages should be English
+        assert self.source_language == "en" or self.target_language == "en"
+
+    def _deduplicate(self, dataset: List):
+        """
+        Remove instances in the dataset with the same label.
+        """
+
+        deduplicated_dataset = []
+        seen_labels = set()
+        for example in dataset:
+            if example[self.target_language] not in seen_labels:
+                seen_labels.add(example[self.target_language])
+                deduplicated_dataset.append(example)
+        return deduplicated_dataset
+
+    def get_instances(self) -> List[Instance]:
+        hlog("Loading the HuggingFace dataset. The first time could take several minutes.")
+        subset_name = f"{self.source_language if self.source_language!='en' else self.target_language}-en"
+        hf_dataset: Any = load_dataset("wmt14", subset_name)
+        splits = {"train": TRAIN_SPLIT, "validation": VALID_SPLIT, "test": TEST_SPLIT}
+
+        instances: List[Instance] = []
+        hlog("Generating instances")
+        # Some training sets are too large, so we will only take a random subset of it.
+        hf_dataset["train"] = hf_dataset["train"].shuffle(seed=42)[:MAX_TRAIN_INSTANCES]
```

Comment:
How big are the training sets?

Reply:
It differs among language pairs (subsets):

cs-en: 953,621
de-en: 4,508,785
fr-en: 40,836,715
hi-en: 32, 863
ru-en: 1,486,965